src/Components/Publishing/Layouts/StandardLayout.tsx

```diff
@@ -92,7 +92,7 @@ export class StandardLayout extends React.Component<
   renderTopRailDisplayAd(isMobileAd: boolean) {
     const { areHostedAdsEnabled, article } = this.props
     const adDimension = isMobileAd
-      ? AdDimension.Mobile_TopLeaderboard
+      ? AdDimension.Mobile_InContentMR1
```

Comment:
What does MR stand for?

Reply:
It is language specific to Hashtag Labs, that's the string we have to include in the HTML so on their end they know what size ad to render, and on what page. 

Their documentation outlines some of that https://htlbid.com/clients/artsy/index.html, and I'm working on some for Artsy internal use as well.